panel/util/__init__.py

```diff
@@ -466,3 +466,9 @@ def styler_update(styler, new_df):
         todo = tuple(ops)
         todos.append(todo)
     return todos
+
+
+def try_datetime64_to_datetime(value):
+    if hasattr(value, "astype"):
```

Comment:
I guess my main question is why you can't explicitly check for a datetime64 value? This seems unsafe.

Reply:
Oh yeah; I forget we import numpy.